src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/RunWorkflowTests.kt

```diff
@@ -221,4 +223,24 @@ class RunWorkflowTests : SeleniumTest()
         assertThat(branchSelect.getAttribute("value")).isEqualTo("other")
         assertThat(commitSelect.getAttribute("value")).isNotBlank()
     }
+
+    @Test
+    fun `can select workflow progress tab and selecting a workflow option generates reports table`()
+    {
+        val link = driver.findElement(By.id("workflow-progress-link"))
+        assertThat(link.text).isEqualTo("Workflow progress")
+        link.click()
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.id("workflow-progress-tab")))
+        val vSelectInput = driver.findElement(By.tagName("input"))
+        vSelectInput.sendKeys("workf")
+
+        val vSelect = driver.findElement(By.id("workflows"))
+        val dropdownMenu = vSelect.findElements(By.tagName("li"))
+        assertThat(dropdownMenu[0].text).contains("workflow1\n" +
+            "Tue Jun 15 2021, 14:50")
+        dropdownMenu[0].click()
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.id("workflow-table")))
+        val table = driver.findElement(By.id("workflow-table"))
+        assertThat(table.text).contains("Reports")
```

Comment:
@EmmaLRussell this should contain the reports in the table, not just the header. I cannot get them to render. I tried inserting a report with the helper into the database but it isn't showing up. Do you have any suggestions?

Reply:
I think the problem is that, although we've told our database tables about the workflow in the setup method, orderly.server doesn't know about it, so we're not going to get a sensible response from the status endpoint. So we need to run a workflow for real - can't do this through the UI until the Submit ticket is done, but for now we can invoke some javascript with Mark's cool fetch script. 

This works for me, I'll leave you to add any further tests of the table you want